bin/test_staging_update_census_mapbox

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+
+# The map on code.org/yourschool is run off of Mapbox.
+# This script updates the data on Mapbox by querying all rows from
+# census_summaries table for the appropriate school year,
+# creating maptiles with Mapbox's command line tool, Tippecanoe,
+# and uploading them to Mapbox's server.
+
+require_relative '../dashboard/config/environment'
+require_relative '../pegasus/helpers/properties'
+
+require_relative './cron/upload_to_mapbox'
+
+raise "CDO.mapbox_upload_token is not defined" unless CDO.mapbox_upload_token
+
+CSV_IMPORT_OPTIONS = {col_sep: ",", headers: true, encoding: 'bom|utf-8'}.freeze
+
+def make_location(lat, long)
+  return nil if lat.nil? || long.nil?
+
+  "#{format('%.6f', lat)}, #{format('%.6f', long)}"
+end
+
+def main
+  CDO.log.info "Loading census summary data."
+  AWS::S3.seed_from_file('cdo-census', "access-report-data-files-2024/final-csv/final_csv_2024_09_30.csv") do |filename|
+    year = '2024'
+    geojson = JSON.parse('{ "type": "FeatureCollection", "features": [] }')
+    entries_skipped = 0
+
+    CSV.read(filename, **CSV_IMPORT_OPTIONS).each do |row|
+      # Parse or skip row data
+      parsed = row.to_hash.symbolize_keys
+      school = School.find_by_id(parsed[:school_id])
+
+      if school.nil?
+        entries_skipped += 1
+        next
+      end
+
+      teaches_cs = parsed[:teaches_cs] == 'unknown' ? 'U' : parsed[:teaches_cs]
+
+      # Add parsed row to map
+      geojson["features"] <<
+        {
+          geometry: {
+            coordinates: [school.longitude.to_f, school.latitude.to_f],
+            type: "Point"
+          },
+          properties: {
+            school_id: school.id,
+            year: year,
+            school_name: school.name.titleize,
+            school_city: school.city.titleize,
+            school_state: school.state,
+            teaches_cs: teaches_cs
+          },
+          type: "Feature"
+        }
```

Comment:
Add each row from the `.csv` directly to `geojson["features"]` (mirrored from [logic in update_census_mapbox](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/update_census_mapbox#L27-L42)) rather than seeding `CensusSummaries` using the `.csv` then populating `geojson["features"]` from `CensusSummaries`